### ISSUE TITLE
Defaults for moench03.2

### ISSUE THREAD

**Issue by anberga** [slsdetectorgroup/slsDetectorPackage#783]
##### *Detector type: 

Moenc03.2

##### *Software Package Version: 

developer
##### *State the feature:

Setup the server and receiver such that these function are executed (or equivalent)

clearbit 0x7f 1

dac vb_comp 1300
dac vdd_prot 1000 
dac vin_com 1400
dac vref_prech 680 
dac vb_pixbuf 1428
dac vb_ds 1200
dac vref_ds 800 
dac vref_comp 900

readoutspeed half_speed

#this would be adcpipeline
reg 0x42 0x12 
adcphase 0 deg

#adcvpp
adcreg 0x18 0x4
rx_discardpolicy discardpartial
rx_framesperfile 100000

**Comment by anberga** [comment]
- change default DAC names and DAC values such that they match the old ones
OLD (correct names and values)
dacvalues [vbp_colbuf 1300, vipre 1000, vin_cm 1400, vb_sda 680, vcasc_sfp 1428, vout_cm 1200, vipre_cds 800, ibias_sfp 900]

NEW (left over from JF)
[vb_comp 1220, vdd_prot 3000, vin_com 1053, vref_prech 1450, vb_pixbuf 750, vb_ds 1000, vref_ds 480, vref_comp 420]

**Comment by anberga** [comment]
in full speed 
reg 0x42 0x12
adcphase 300 deg

half speed and quarter speed
reg 0x42 0x12
adcphase 0 deg

**Comment by kozlowp1** [comment]
I will just leave it here so that all things are together. I will take care of it when I am back in August.

"I was trying to do some speed tests with moench to see how fast we can run
In theory, with 40Mhz clock, 5000 samples/ADC I calculate 125us
Of course there will be a bit of overhead, but OK.
So, I tried non-parallel mode and put exptime 0 and period 0
The progress bar messed up, but at the end it looked like it had run through and the total time was 14s/10000 frames
So I thought that an exposure time of 140us in parallel mode should work.
I configured it, but it froze.
Ater that I don't get packets any longer even if I increase the exposure time."

"In software, for exptime, it always reduces 2 clock cycles. If it becomes less than 0, then it is set to 0.
For period, the number of clocks are saved as it is without any further change."
